server/views/temporary-accommodation/bookings/history.njk

```diff
@@ -0,0 +1,40 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/identity-bar/macro.njk" import mojIdentityBar -%}
+{% from "../components/booking-info/macro.njk" import bookingInfo %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Booking history" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: paths.bookings.show({ premisesId: premises.id, roomId: room.id, bookingId: booking.id })
+  }) }}
+
+  {% include "../../_messages.njk" %}
+
+  <h1>Booking history</h1>
+
+  <div class="location-header">
+    <p><span class="govuk-!-font-weight-bold">CRN:</span> {{ booking.person.crn }}</p>
+
+    <h2>Bedspace reference</h2>
+    <p>{{ room.name }}</p>
+
+    <h2>Property address</h2>
+    <p>{{ premises.addressLine1 }}<br />{{ premises.postcode }}</p>
+  </div>
+
+  {% for historicBookingDetail in history | reverse %}
```

Comment:
If I'm understanding things correctly we reverse the list in `deriveBookingHistory` only to then reverse it once more here. Could we do without reversing it at all?

Reply:
Heh, that's how I originally had it (with the extensions also in reverse order), but then I felt icky about having to just *know* that's the ordering, and it felt more natural for the fact that it's displayed in reverse order to be a template concern

It also makes the E2E tests easier - just pushing a booking/extension onto the array we're checking the page against, as we update the booking